### PR TITLE
Add a project feature handling

### PR DESCRIPTION
**Pull request by kenpham4real**

### CODE REVIEW

File: src/pages/app/posting_project_user/CreatePostModal_3.js [hunk 1]
```diff
@@ -42,6 +42,26 @@ const CreatePostModal_3 = (props) => {
     projectCategory,
   } = props.location;
 
+  const _onContinue_3 = () =>{
```
Comment (kenpham4real) [inline src/pages/app/posting_project_user/CreatePostModal_3.js:45]: Such a BAD and UNCLEAR naming. Please rename it @TrNgTien

File: src/store/actions/posting-project-user/project/project.js [hunk 2]
```diff
@@ -129,54 +129,70 @@ export const _createProject_ppu = (
...
-  imageFIle,
+  imageFile,
   questions,
   category = "General"
 ) => {
+  //Handle data error
+  if (orgId == "") orgId = JSON.parse(localStorage.getItem("userData")).userId;
+  if (!deadline) deadline = "";
+
+  //For debugging purpose
```
Comment (kenpham4real) [inline src/store/actions/posting-project-user/project/project.js:140]: We're working in production. No longer debugging here. @Chillisaucery

File: src/store/actions/posting-project-user/project/project.js [hunk 3]
```diff
@@ -129,54 +129,70 @@ export const _createProject_ppu = (
...
+    const projectImageUrl = await _imageUploadHandler(imageFile);
     console.log("projectImageUrl", projectImageUrl);
-    const projectRef = firebase_db
-      .collection("public-projects")
-      .doc(`${projectId}`);
-    const organizationRef = firebase_db
-      .collection("organization")
-      .doc(`${orgId}`);
+    const projectRef = firebase_db.collection("public-projects").doc(projectId);
+    const organizationRef = firebase_db.collection("organization").doc(orgId);
```
Comment (kenpham4real) [inline src/store/actions/posting-project-user/project/project.js:161]: Wrong syntax. 
```suggestion
    const projectRef = firebase_db.collection("public-projects").doc(`${projectId}`);
    const organizationRef = firebase_db.collection("organization").doc(`${orgId});
```

Comment (TrNgTien) [reply src/store/actions/posting-project-user/project/project.js]: image

Why with this one it's correct

**Pull request merged**